guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sum-all-odd-fibonacci-numbers/index.md

```diff
@@ -88,11 +88,8 @@ As you get the next odd one, don't forget to add it to a global variable that ca
             arrFib.unshift(nextFib);
         }
 
-        // Sum only the odd numbers and return the value
-        // First, reverse the array to avoid starting acc with the first/greater number when it's even
-        return arrFib.reverse().reduce((acc, curr) => {
-            return acc + curr * (curr % 2);
-        }, 0);
+        // We return the array, filtering and summing (reducing) odd numbers at once. 
```

Comment:
```suggestion
        // We filter the array to get the odd numbers and reduce them to get their sum.
```
I wasn't sure about the original wording as it sounded like we were returning the array itself.